website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -0,0 +1,275 @@
+  * Karpenter now adds a karpenter.sh/unregistered taint to nodes in injected UserData when using alias in AMISelectorTerms or non-Custom AMIFamily. When using AMIFamily: Custom, users will need to add this taint into their UserData, where Karpenter will automatically remove it when provisioning nodes. 
+* API Moves: 
+  * ExpireAfter has moved from the `NodePool.Spec.Disruption` block to `NodePool.Spec.Template.Spec`, and is now a drift-able field.
+  * KubeletConfig was moved to the EC2NodeClass from the NodePool.
+* RBAC changes: added `delete pods` | added `get, patch crds` | added `update nodes` | removed `create nodes`
+* Breaking API (Manual Migration Needed): 
+  * Ubuntu is dropped as a first class supported AMI Family
+  * `karpenter.sh/do-not-consolidate` (annotation), `karpenter.sh/do-not-evict` (annotation), and `karpenter.sh/managed-by` (tag) are all removed. `karpenter.sh/managed-by`, which currently stores the cluster name in its value, will be replaced by eks:eks-cluster-name
+  * The taint used to mark nodes for disruption and termination changed from `karpenter.sh/disruption=disrupting:NoSchedule` to `karpenter.sh/disrupted:NoSchedule`. It is not recommended to tolerate this taint, however, if you were tolerating it in your applications, you'll need to adjust your taints to reflect this.
+* Environment Variable Changes: 
+  * Environment Variable Changes
+  * LOGGING_CONFIG, ASSUME_ROLE_ARN, ASSUME_ROLE_DURATION Dropped
+  * LEADER_ELECT renamed to DISABLE_LEADER_ELECTION
+  * DRIFT_ENABLED dropped and promoted to Stable, cannot be disabled.
+      * Users currently opting out of drift, disabling the drift feature flag will no longer be able to do so.
+* Defaults changed: 
+  * API: Karpenter will drop support for IMDS access from containers by default on new EC2NodeClasses by updating the default of httpPutResponseHopLimit from 2 to 1.
+  * API: ConsolidateAfter is required. Users couldn’t set this before with ConsolidationPolicy: WhenUnderutilized, where this is now required. Users can set it to 0 to have the same behavior as in v1beta1. 
+  * API: NodeClassRef fields are now all required, and apiVersion has been renamed to group
+  * API: AMISelectorTerms are required. Setting an Alias cannot be done with any other type of term, and must match the AMI Family that's set or be Custom.
+  * Helm: Deployment spec TopologySpreadConstraint to have required zonal spread over preferred. Users who had one node running their Karpenter deployments need to either:
+    * Have two nodes in different zones to ensure both Karpenter replicas schedule
+    * Scale down their Karpenter replicas from 2 to 1 in the helm chart
+    * Edit and relax the topology spread constraint in their helm chart from DoNotSchedule to ScheduleAnyway
+  * Helm: controller.METRICS_PORT default changed back to 8080
```

Comment:
This isn't just helm specific. This was also a default that was changed generally in the binary as well